web_ui/packages/ui/src/toggle-buttons/toggle-buttons.component.tsx

```diff
@@ -21,6 +21,7 @@ const ToggleButton = <T extends string>({
 }: ToggleButtonProps<T>) => {
     return (
         <Button
+            data-activated={selectedOption === option}
```

Comment:
Alternatively we might also be able to use `aria-selected` or `aria-pressed`, or maybe use [ToggleButton](https://react-spectrum.adobe.com/react-spectrum/ToggleButton.html)?